tpu_inference/distributed/tpu_connector_local.py

```diff
@@ -725,6 +752,13 @@ def __init__(self, vllm_config: VllmConfig, connector: "TPUConnector"):
 
         self.runner: Optional[TPUModelRunner] = None
         self.mesh: Optional[Mesh] = None
+        self.swap_op_type = os.getenv("TPU_KV_OFFLOADING_SWAP_OP_TYPE",
```

Comment:
for uniformity, let's use the existing prefix used in other env variables TPU_OFFLOAD_*